packages/wonder-blocks-tooltip/src/components/tooltip-tail.js

```diff
@@ -247,24 +247,6 @@ export default class TooltipTail extends React.Component<Props> {
         ];
     }
 
-    _minDistanceFromCorners(placement: Placement): number {
-        const minDistanceFromCornersForTopBottom = Spacing.medium_16;
-        const minDistanceFromCornersForLeftRight = 7;
-
-        switch (placement) {
-            case "top":
-            case "bottom":
-                return minDistanceFromCornersForTopBottom;
-
-            case "left":
-            case "right":
-                return minDistanceFromCornersForLeftRight;
-
-            default:
-                throw new Error(`Unknown placement: ${placement}`);
-        }
-    }
```

Comment:
**praise:** I didn't realize that this code wasn't being used in production.  Thanks for removing it.